proprietary/Components/src/denhaag/form-field-description.tokens.json

```diff
@@ -0,0 +1,16 @@
+{
+  "denhaag": {
+    "form-field": {
+      "description": {
+        "color": { "value": "{denhaag.color.grey.4}" },
+        "font-family": { "value": "{utrecht.document.font-family}" },
+        "font-size": { "value": "{denhaag.typography.scale.s.font-size}" },
+        "font-weight": { "value": "{denhaag.typography.weight.regular}" },
+        "line-height": { "value": "{denhaag.typography.scale.s.line-height}" },
+        "error": {
+          "color": { "value": "{denhaag.color.red.3}" }
+        }
+      }
+    }
+  }
+}
```

Comment:
This file can be removed in favour of the one in `utrecht/`

Reply:
Den Haag needs these values and I removed it from `form-field` to have the same structure as for utrecht. I can put the description back as it was in `form-field`.